docs/FAQ.md

```diff
@@ -0,0 +1,124 @@
+
+### Do I need a GPU to run NNCF?
+Currently all NNCF-supported backends allow running in a CPU-only mode, and NNCF does not disturb this.
+Note, however, that training-aware compression will naturally work much slower on most CPUs when compared with GPU-powered execution.
+
+Check out the [notebooks](https://github.com/openvinotoolkit/openvino_notebooks#-model-training) for examples of NNCF being applied on smaller datasets which work in a reasonable amount of time on a CPU-only setup.
+
+### NNCF supports both training and post-training compression approaches, how do I know which I need?
+The rule of thumb is - start with post-training compression, and use training compression if you are not satisfied with the results and if training compression is possible for your use case.
+Post-training is faster, but can degrade accuracy more than the training-enabled approach.
+
+### I don't see any improvements after applying the `*_sparsity` algorithms
+The sparsity algorithms introduce unstructured sparsity which can only be taken advantage of in terms of performance by using specialized hardware and/or software runtimes. Within the scope of these algorithms, NNCF provides functionally correct models with non-salient weights simply zeroed out, which does not lead to the reduction of the model checkpoint size. The models can, however, be used for benchmarking experimental/future hardware or runtimes, and for SOTA claims of applying unstructured sparsity on a given model architecture.
+
+For an opportunity to observably increase performance by omitting unnecessary computations in the model, consider using the [filter pruning](./compression_algorithms/Pruning.md) algorithm. Models compressed with this algorithm can be executed more efficiently within OpenVINO Inference Engine runtime when compared to the uncompressed counterparts.
+
+### What is saturation issue and how to avoid it?
+On older generations of Intel CPUs (those not supporting [AVX-VNNI](https://en.wikipedia.org/wiki/Advanced_Vector_Extensions#AVX-VNNI)) convolutions and linear layer INT8 execution is implemented in OpenVINO's Inference Engine in such a way that mathematical overflow manifests itself _if more than 128 levels are used in the quantized domain_ (out of possible 2^8 = 256) for the weights of the corresponding operations.
+This is referred to as "saturation issue" within NNCF.
+On newer AVX-VNNI enabled Intel CPUs the Inference Engine uses a better set of instructions that do not exhibit this flaw.
+
+For this reason, and to support best compatibility across Intel CPUs, the weights of the convolutional and linear operations in the DL models are quantized by NNCF to only utilize 128 levels out of possible 256, effectively applying 7-bit quantization.
+This does not apply to activation quantization - values of quantized activation tensors will utilize the entire set of quantization levels (256) available for INT8. 
```

Comment:
Could you add here that this can be disabled (and how, or with a link to instructions?).

Reply:
Done